dist/api_data.py

```diff
@@ -450,6 +450,18 @@ def __ge__(self, other):
         for space to be available in cache before giving up. Default will
         wait forever''',
         min=0),
+    Config('cache_overflow', '', r'''
+        cache overflow configuration options''',
+        type='category', subconfig=[
+        Config('file_max', '0', r'''
+            The maximum number of bytes that WiredTiger is allowed to use for
+            its cache overflow mechanism. If the cache overflow file exceeds
+            this size, a panic will be triggered. The default value means that
+            the cache overflow file is unbounded and may use as much space as
+            the filesystem will accommodate. The minimum non-zero setting is
+            100MB.''',    # !!! Must match WT_LAS_FILE_MIN
+            min='0')
```

Comment:
Because we want to be able to specify 0 (unbounded).
We have validation in `__wt_las_config` for the 100MB check.

This is the same pattern we follow for the `io_capacity` config.